src/app/routes/liveRadio/getInitialData/index.test.js

```diff
@@ -38,19 +38,25 @@ describe('Get initial data for live radio', () => {
 
   it('should override renderer on test', async () => {
     process.env.SIMORGH_APP_ENV = 'test';
-    await getInitialData({ path: 'mock-live-radio-path', pageType });
+    await getInitialData({
+      path: 'mock-live-radio-path',
+      pageType: MEDIA_PAGE,
+    });
     expect(spy).toHaveBeenCalledWith({
       path: 'mock-live-radio-path?renderer_env=live',
-      pageType,
+      pageType: 'media',
```

Comment:
Is this meant to be `MEDIA_PAGE`?

Reply:
It's part of the expect block so it shouldn't be a const? If the const changes then this test will just always pass. Or is it because we're not testing that specifically then we shouldn't care?

Reply:
`MEDIA_PAGE` is a constant whose value is and will always be `media`.
If someone decides that value should change, they can update it in `pageTypes.js` and it will update everywhere `MEDIA_PAGE` is being used. This means that there is a single source of truth, but only if it's used and here it isn't.

Reply:
In that case do we need to even have the `pageType` check on these tests?

Reply:
I'm not sure as I didn't write these tests. Does `getInitialData` require `pageType`? I would imagine it does.

Reply:
Yeah it does. Bit of a weird side effect as it looks like we're testing it but we're not, we just want to know the renderer override is correct. I shall amend.